### DISCUSSION TITLE
Mixing chip-tool and Alexa-hosted node-matter

### DISCUSSION THREAD

**Discussion by javierbarellano4v** [mfucci/node-matter#209] [Q&A] [answered]
So, I've previously setup node-matter in Alexa environment (Echo Dot) using Alexa app on Android. Paired and interact via Echo Dot with "light bulb".

When, I brought up the chip-tool on the same network, I was expecting to see node-matter when I run 'discover'.

**Comment by Apollon77** [comment, marked as answer]
When node-matter is paired with Alexa then you need to use Alexa to get a "new pairing code" and THEN it should show up

**Comment by javierbarellano4v** [comment]
Is this necessary because chip-tool is a matter controller?

**Reply by Apollon77** [reply]
Matter devices pair with multiple controllers but the first controller is kind of the "main one" (simplified) SO as soon as a device is paired it is not announced as "unpaired" - unless you enable the commissioning by telling the first controller that you need a new pairing code ... then the device is announcing itself and a pairing code displayed by that first controller can be used for pairing.

**Comment by javierbarellano4v** [comment]
I should've been able to answer my initial question...., I've asked this question before regarding multi-admin..., forgot that chip-tool is a controller.

**Reply by Apollon77** [reply]
:-)
